mapie/conformity_scores.py

```diff
@@ -138,27 +118,53 @@ def check_consistency(self, y: NDArray, y_pred: NDArray) -> None:
         ValueError
             If the two methods are not consistent.
         """
-        if self.consistency_check:
-            conformity_scores = self.get_signed_conformity_scores(y, y_pred)
-            abs_conformity_scores = np.abs(
-                self.get_estimation_distribution(y_pred, conformity_scores) - y
+        score_distribution = self.get_estimation_distribution(
+            y_pred, conformity_scores
+        )
+        abs_conformity_scores = np.abs(np.subtract(score_distribution, y))
+        max_conf_score = np.max(abs_conformity_scores)
+        if max_conf_score > self.eps:
+            raise ValueError(
+                "The two functions get_conformity_scores and "
+                "get_estimation_distribution of the ConformityScore class"
+                " are not consistent. "
+                "The following equation must be verified: "
+                "self.get_estimation_distribution(y_pred, "
+                "self.get_conformity_scores(y, y_pred)) == y. "  # noqa: E501
+                f"The maximum conformity score is {max_conf_score}."
+                "The eps attribute may need to be increased if you are "
+                "sure that the two methods are consistent."
             )
-            max_conf_score = np.max(abs_conformity_scores)
-            if max_conf_score > self.eps:
-                raise ValueError(
-                    "The two functions get_conformity_scores and "
-                    "get_estimation_distribution of the ConformityScore class "
-                    "are not consistent. "
-                    "The following equation must be verified: "
-                    "self.get_estimation_distribution(y_pred, self.get_conformity_scores(y, y_pred)) == y. "  # noqa: E501
-                    f"The maximum conformity score is {max_conf_score}."
-                    "The eps attribute may need to be increased if you are "
-                    "sure that the two methods are consistent."
-                )
+
+    def get_conformity_scores(
+        self, y: ArrayLike, y_pred: ArrayLike
+    ) -> NDArray:
+        """
+        Get the conformity score considering the symmetrical property if so.
+
+        Parameters
+        ----------
+        y : NDArray
+            Observed values.
+        y_pred : NDArray
+            Predicted values.
+
+        Returns
+        -------
+        np.ndarray
+            Conformity scores.
+        """
+        conformity_scores = self.get_signed_conformity_scores(y, y_pred)
+        if self.consistency_check:
+            self.check_consistency(y, y_pred, conformity_scores)
+        if self.sym:
+            conformity_scores = np.abs(conformity_scores)
```

Comment:
Why should we add an argument self.sym? May np.abs() be directly included into self.get_signed_conformity_scores ?

Reply:
Done.